### DISCUSSION TITLE
Open directly the router settings

### DISCUSSION THREAD

**Discussion by devemux86** [devemux86/cruiser#174] [Q&A] [answered]
Let's make our life easier and quickly access each router's settings without going into settings! :slightly_smiling_face: 

We can add a menu that will open directly the settings of the selected / active router:
1. In menu "Routing"
2. In route actions (long press on route)
3. They open directly e.g. BRouter settings

Or do you prefer to open directly the vehicle settings of the router?
e.g. BRouter > Motorcycle settings

If you need additional features or other settings to open directly from the map screen, please let me know!

<img src="https://github.com/devemux86/cruiser/assets/3484020/00cf45bf-6f25-4b22-ad66-747de59974da" width="33%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/486fe36b-7f7a-48f4-bafb-cdb2d2017f46" width="33%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/d2665218-6801-485e-a650-64b6041eb24b" width="33%"/>

**Comment by WolfgangZ42** [comment]
Just great! I like the combination of Car/Motorcycle - avoids questions. And this could make own .brf-files obsolete. An important step in the right direction!
I miss the "Segment profile" in the selection ...

**Reply by devemux86** [reply]
BRouter was not active when taking the screenshot, I updated the image!

**Comment by devemux86** [comment, marked as answer]
Open directly the router settings is available in [Cruiser 3.0.20](https://play.google.com/store/apps/details?id=gr.talent.cruiser).

- In menu "Routing"
- In route actions (long press on route)

See the images here: https://github.com/devemux86/cruiser/discussions/174#discussion-5215194
